diff --git a/variables.ini.dist b/variables.ini.dist
index cc413e7b..9e10958e 100644
--- a/variables.ini.dist
+++ b/variables.ini.dist
@@ -72,10 +72,13 @@ LOGSTASH_HOST = # logstash.server.com
 LOGSTASH_PORT = # 8000
 METRIC_SERVER = # atmosphere-metrics.server.com
 SELF_SIGNED_CERT =  False # Boolean required
-TESTING =  False # Boolean required
+TESTING =  True # Boolean required
+TEST_RUNNER_USER = # Nothing
+TEST_RUNNER_PASS = # Nothing
+DJANGO_JENKINS = False
 #local.py - site configuration
 AUTO_CREATE_NEW_ACCOUNTS =  False # Boolean required
-USE_JETSTREAM_PLUGIN =  False # Boolean required
+USE_JETSTREAM_PLUGIN =  True # Boolean required
 TIME_ZONE = # America/Phoenix
 SITE_NAME = # CyVerse
 ORG_NAME = # CyVerse
